utils/nctl/sh/scenarios/accounts_toml/itst01_private_chain.accounts.toml.override

```diff
@@ -43,6 +43,12 @@ balance = "0"
 bonded_amount = "1000000000000005"
 delegation_rate = 5
 
+[[delegators]]
+validator_public_key = "PBK_V5"
+delegator_public_key = "020248509e67db3127f82d5224c5c18eac00f96d1edeadbadc8eb2c8606227b56873"
```

Comment:
It's unclear who the delegator is here. We should use symbolic names in the overrides e.g. `PBK_U1`.

Reply:
Fixed, along with other necessary changes, in https://github.com/casper-network/casper-node/pull/4343/commits/f6af9d9b623b9669f92398c2ab14b105abc834d0